toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -550,6 +546,27 @@ os:
     corruptionOption: panic
 ```
 
+The Verity-enabled root filesystem is always mounted as read-only. Its root hash
+and hash tree are computed at build time and verified by systemd during the
+initramfs phase on each boot. When enabling the Verity feature, it is
+recommended to create a writable persistent partition (e.g., /var) for any
+directories that require write access. Critical files and directories, such as
+SSH host keys or logs, can be redirected to the writable partition using
+symlinks or similar methods. See [Verity Filesystem Layout
```

Comment:
why not call out `overlays` here as well?

Reply:
My interpretation of Chris' comment was that it shouldn't be the sole recommendation but IMO we should include it as one possible option.

Reply:
It makes sense to me.